.doks/doks-cluster/locals.tf

```diff
@@ -0,0 +1,281 @@
+######################
+# values_dbs_dev.yaml
+######################
+
+locals{
+  template_values_dbs_dev = <<EOF
+global:
```

Comment:
can't we just `sed` this on the `values-production.yaml`? 

The user,password, host and port should be the same for all dbs, right?

Reply:
I see your point but we use the .doks/values.yaml for our digitalocean environments. There are different settings there such are component resources or persistence sizes on values-production.yaml.

As for not using `sed`, I am changing that now. :)

Reply:
Actually, now i remember why!
If you use a local_exec, it will be saved in the state. Which means that the next time you run the terraform apply, that specific resource will not be created. Therefore, you will not have your templated file in the local filesystem on the build directory.